moveit_ros/planning/moveit_cpp/src/planning_component.cpp

```diff
@@ -57,15 +57,6 @@ PlanningComponent::PlanningComponent(const std::string& group_name, const MoveIt
     RCLCPP_FATAL_STREAM(LOGGER, error);
     throw std::runtime_error(error);
   }
-  plan_request_parameters_.load(node_);
```

Comment:
`plan_request_parameters_` was only used by `plan()` so I moved this code directly into the function and removed the member variable